api/vote.go

```diff
@@ -2,19 +2,57 @@ package api
 
 import (
 	"bytes"
+	"encoding/hex"
 	"encoding/json"
 	"net/http"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/go-chi/chi/v5"
 	"github.com/vocdoni/vocdoni-z-sandbox/circuits"
 	"github.com/vocdoni/vocdoni-z-sandbox/circuits/ballotproof"
 	"github.com/vocdoni/vocdoni-z-sandbox/crypto/signatures/ethereum"
+	"github.com/vocdoni/vocdoni-z-sandbox/log"
 	"github.com/vocdoni/vocdoni-z-sandbox/storage"
 	"github.com/vocdoni/vocdoni-z-sandbox/types"
+	"github.com/vocdoni/vocdoni-z-sandbox/util"
 )
 
-// newVote creates a new voting process
-// POST /vote
+// voteStatus returns the status of a vote for a given processID and voteID
+// GET /votes/status/{processId}/{voteId}
```

Comment:
the `voteID` as the hash of the ballot proof inputs, already contain the `processID`. Is it required for any reason?

Reply:
Because we index by processId to be able to perform afterward the prune (so we remove based on prefix)